gcc/rust/resolve/rust-early-name-resolver-2.0.h

```diff
@@ -28,6 +28,8 @@
 namespace Rust {
 namespace Resolver2_0 {
 
+using ResolveError = std::function<void ()>;
```

Comment:
I think we should simply store errors in an `Error` container like the parser does. I like the lambda idea, but they are unwieldy in C++

Reply:
Fixed in fdccd48aff5.